core/src/main/scala/kafka/server/DelegationTokenManagerZk.scala

```diff
@@ -292,6 +298,8 @@ class DelegationTokenManagerZk(config: KafkaConfig,
               expireResponseCallback(Errors.NONE, now)
             } else if (tokenInfo.maxTimestamp < now || tokenInfo.expiryTimestamp < now) {
               expireResponseCallback(Errors.DELEGATION_TOKEN_EXPIRED, -1)
+            } else if (now > Long.MaxValue - expireLifeTimeMs) {
```

Comment:
we should align the behavior of handling the overflow timestamp, right?

Reply:
Yes, I missed it.
Thanks for pointing out 😸